### DISCUSSION TITLE
Point-in-time entries

### DISCUSSION THREAD

**Discussion by tecunningham** [jotaen/klog#46] [Feature ideas & discussions]
It might be nice for the KLOG-format to allow entries just at a single point in time, with no duration, e.g.:

```klog
2018-03-24
First day at my new job
    8:30 - 17:00
    12:00 #workout 
```

You can already have zero-duration entries, like 12:00-12:00, but this format might be a bit more natural. Some use-cases:

- Keeping track of doing exercise, when duration doesn't really matter
- Keeping track of meals
- Keeping track of commits

**Comment by vladdeSV** [comment]
I keep track of single events with 
```
    0m <activity>
```

**Comment by jotaen** [comment]
Interesting use-case, I haven’t thought of this at all yet.

At the moment I’d also suggest using a workaround like @vladdeSV suggests, e.g.

```
2000-01-01
    0m Something
    12:00-12:00 Something
```

Maybe that is sufficient already? Otherwise I’d be generally open for extending the spec, should this be a common thing that people want to do. The tricky thing is probably to come up with a nice and robust syntax for it. (Nice syntax would mean something that uses symbolic characters, not keywords, and that is still sufficiently intuitive to understand.)

Happy to leave this ticket open, so we can collect ideas or discuss further.

**Reply by thiloplanz** [reply]
A comment syntax could be used for this (allowing arbitrary remarks that are ignored by the program), https://github.com/jotaen/klog/discussions/54

```
2018-03-24
First day at my new job
    8:30 - 17:00
    ; 12:00 #workout 
```
